ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml

```diff
@@ -349,3 +349,48 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )vendor,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
We could make this optionally run only when that file changes?

Reply:
Yeah great, I actually missed that I could do that, i'll make the change

Reply:
`pkg/apis/integreatly/v1alpha1/rhmi_types.go`

Reply:
see https://github.com/openshift/release/blob/master/ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml#L17

Reply:
https://github.com/openshift/release/blob/master/ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml#L4